example/generator/odp_generator.c

```diff
@@ -1733,7 +1822,13 @@ static void usage(char *progname)
 	       "Optional OPTIONS\n"
 	       "  -h, --help       Display help and exit.\n"
 	       "  -e, --srcport src udp port\n"
```

Comment:
What about "src udp start port"?

Reply:
I am not a fan of abbreviation. It is not part of this patch but I guess we can make it more verbose in next version.

Reply:
@bogdanPricope For me it is a problem of having an "end", but not having "start" corresponding to that "end".

Reply:
Yes, this is what I expected. Thank you!